docs/design/coreclr/profiling/davbr-blog-archive/Profiler stack walking Basics and beyond.md

```diff
@@ -145,77 +145,77 @@ But before you get too deep, note that the issue of whether and how to seed a st
 
 For the truly adventurous profiler that is doing an asynchronous, cross-thread, seeded stack walk while filling in the unmanaged holes, here’s what it would look like.
 
-| 
+|
 
-Block of  
-Unmanaged  
+Block of
+Unmanaged
 Frames
 
- | 
-1. You suspend the target thread (target thread’s suspend count is now 1) 
-2. You get the target thread’s current register context 
-3. You determine if the register context points to unmanaged code (e.g., call ICorProfilerInfo2::GetFunctionFromIP(), and see if you get back a 0 FunctionID) 
+ |
+1. You suspend the target thread (target thread’s suspend count is now 1)
+2. You get the target thread’s current register context
+3. You determine if the register context points to unmanaged code (e.g., call ICorProfilerInfo2::GetFunctionFromIP(), and see if you get back a 0 FunctionID)
 4. In this case the register context does point to unmanaged code, so you perform an unmanaged stack walk until you find the top-most managed frame (D)
  |
-| 
+|
 
-Function D  
+Function D
 (Managed)
```

Comment:
Same as before, if the intention here is to make these in separate lines, they should be:

```suggestion
Function D\
(Managed)
```

Reply:
Either this, or make it a code block. It would be nice to fix all instances of this pattern the same way.